src/Components/ADT3DSceneBuilder/Internal/Behaviors/Widgets/WidgetBuilders/DataHistoryWidgetBuilder/Internal/TimeSeriesFormCallout.tsx

```diff
@@ -151,8 +160,20 @@ const TimeSeriesFormCallout: React.FC<IProp> = ({
                     propertyExpression={{
                         expression: seriesToEdit?.expression
                     }}
+                    description={
+                        seriesToEdit?.propertyType &&
+                        !numericPropertyValueTypes.includes(
+                            seriesToEdit.propertyType
+                        )
+                            ? t(
+                                  'widgets.dataHistory.form.timeSeries.nonNumericWarning'
+                              )
+                            : ''
```

Comment:
else condition can be undefined instead of empty string, maybe?

Reply:
agreed